Docs/user_guide/modules/dicom.md

```diff
@@ -155,7 +155,7 @@ Advanced loading (allows loading DICOM data sets using non-default options):
 - **Examine button:** Runs each of the DICOM Plugins on the currently selected series and offers the result in the Loadable items list table.
```

Comment:
Please update the "Plugin selector section" above (no double-arrow, etc.)

Reply:
Could you check, that description is OK?

Reply:
Yes, it is perfect, thank you.